asserts/pool_test.go

```diff
@@ -1716,3 +1716,25 @@ func (s *poolSuite) TestPoolReuseWithClearGroupsAndUnchanged(c *C) {
 	})
 	c.Check(toResolveSeq, HasLen, 0)
 }
+
+func (s *poolSuite) TestBackstore(c *C) {
+	assertstest.AddMany(s.db, s.hub.StoreAccountKey(""), s.dev1Acct)
+	pool := asserts.NewPool(s.db, 64)
+
+	c.Assert(pool.AddUnresolved(s.rev1_1111.At(), "for_one"), IsNil)
```

Comment:
as discussed, if the AtRevision here has Revision: RevisionNotKnown  the test would be slightly clearer. The should be example for that in other tests

Reply:
Yes. done.